eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -1,5 +1,20 @@
 <UsageData>
   <IgnorePatterns>
-    <UsagePattern IdentityGlob="*/*" />
+    <UsagePattern IdentityGlob="Microsoft.SourceBuild.Intermediate.*/*" />
+
+    <!-- These are cases where the component chooses, when built in isolation, to 
+         remain dependent on stable versions. These versions may not have source build intermediates associated with them.
+         They cannot be SBRPs because they actually provide functionality. In full source build, these versions are
+         overridden by previous repo outputs or come from previously source-built artifacts.  --> 
+    <UsagePattern IdentityGlob="Microsoft.CodeAnalysis.Analyzers/*3.3.4*" />
+    <UsagePattern IdentityGlob="Microsoft.CodeAnalysis.AnalyzerUtilities/*3.3.0*" />
+    <UsagePattern IdentityGlob="Microsoft.CodeAnalysis.CSharp.CodeStyle/*4.3.0-1.final*" />
+    <UsagePattern IdentityGlob="Microsoft.CodeAnalysis.VisualBasic.CodeStyle/*4.3.0-1.final*" />
+    <UsagePattern IdentityGlob="Microsoft.Net.Compilers.Toolset/*4.5.0*" />
```

Comment:
I guess one problem I have with this is when is it okay to change these numbers? At some point this validation is goin to land and we're going to get in trouble if we break it. When we do how do we know it's okay to change these numbers? If it's essentially always okay then why have specific versions here versus just using `*` for the version number?

Reply:
I decided that the right thing to do here is to just * these package versions. They will always come from previously source built, and are not a problem as long as roslyn doesn't redist their contents.